pkg/dazzle/build.go

```diff
@@ -544,7 +548,7 @@ func (p *ProjectChunk) buildAsBase(ctx context.Context, dest reference.Named, se
 
 func (p *ProjectChunk) test(ctx context.Context, sess *BuildSession) (ok bool, err error) {
 	if sess == nil {
-		return false, errors.New("Cannot test without a session")
+		return false, errors.New("cannot test without a session")
```

Comment:
Another VSCode/intellij difference - we can discuss how/why our golang setup isn't including golangci-lint later